source/OutfitterPanel.cpp

```diff
@@ -210,17 +210,83 @@ int OutfitterPanel::DetailWidth() const
 
 int OutfitterPanel::DrawDetails(const Point &center)
 {
-	if(!selectedOutfit)
-		return 0;
-	
-	outfitInfo.Update(*selectedOutfit, player, CanSell());
-	Point offset(outfitInfo.PanelWidth(), 0.);
-	
-	outfitInfo.DrawDescription(center - offset * 1.5 - Point(0., 10.));
-	outfitInfo.DrawRequirements(center - offset * .5 - Point(0., 10.));
-	outfitInfo.DrawAttributes(center + offset * .5 - Point(0., 10.));
-	
-	return outfitInfo.MaximumHeight();
+
+	string selectedItem = "Nothing Selected";
+	const Font &font = FontSet::Get(14);
+	const Color &bright = *GameData::Colors().Get("bright");
+	const Color &dim = *GameData::Colors().Get("medium");
+	const Sprite *collapsedArrow = SpriteSet::Get("ui/collapsed");
+
+	int heightOffset = 20;
+
+	if(selectedOutfit)
+	{
+		outfitInfo.Update(*selectedOutfit, player, CanSell());
+		selectedItem = selectedOutfit->Name();
+
+		const Sprite *thumbnail = selectedOutfit->Thumbnail();
+		const Sprite *background = SpriteSet::Get("ui/outfitter selected");
+
+		float tileSize = max(thumbnail->Height(), static_cast<float>(TileSize()));
```

Comment:
Segfault here for outfits with no sprite

Reply:
Fixed with ternary operator, but not tested.

Reply:
![image](https://user-images.githubusercontent.com/20871346/93345747-cb1f5600-f7f8-11ea-8714-9d970091aba4.png)
👍